packages/driver/cypress/integration/e2e/testConfigOverrides.spec.js

```diff
@@ -206,90 +206,87 @@ describe('per-test config', () => {
 
   describe('in nested suite', () => {
     describe('config in suite', {
-      foo: true,
       defaultCommandTimeout: 200,
     }, () => {
-      it('has config.foo', () => {
-        expect(Cypress.config().foo).ok
+      it('has config.defaultCommandTimeout', () => {
         expect(Cypress.config().defaultCommandTimeout).eq(200)
       })
 
       describe('inner suite', {
-        bar: true,
```

Comment:
does `bar: true` throw an error?

Reply:
Yes, because `bar` is not considered a writeable configuration option for per test config

Reply:
hmmm. Are we worried about passivity here? Ideally consumer won't pass configuration options that aren't valid. Currently these will be ignored by Cypress. Throwing an error here would break tests. Would it be worth adding a warning for non-existent config options and in the next MVB throw an error instead? If we want to go ahead & throw the error now we should def note it in the changelog.

Reply:
This is definitely a breaking change and should be targeting the `10.0-release` branch.

Reply:
In product review, it was decided that the behavior for now will be to throw an error only if a user tries to write to a Cypress config property which is not writeable (no error will be thrown for something like `bar: true`). Therefore, this is no longer a breaking change.

Reply:
If a user has `Cypress.config('modifyObstructiveCode', true)` in their test, it currently is benign and will not cause an error, but will throw an error when we release this. Even though it it's not valid, they'll have to change their code for their tests to pass, which makes the change breaking. It's the same reason the [reserved names changes](https://github.com/cypress-io/cypress/pull/18923#discussion_r754577713) are breaking.